src/proof/cnf_proof.cpp

```diff
@@ -135,8 +135,11 @@ void CnfProof::setClauseAssertion(ClauseId clause, Node expr) {
   // case we keep the first assertion. For example asserting a /\ b
   // and then b /\ c where b is an atom, would assert b twice (note
   // that since b is top level, it is not cached by the CnfStream)
-  if (d_clauseToAssertion.find(clause) != d_clauseToAssertion.end())
+  const auto& it = d_clauseToAssertion.find(clause);
```

Comment:
Can u document this null node trick in the comments above this line?

Reply:
Good point, added comment.